<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>FormSite</class>
 <widget class="QWidget" name="FormSite">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>798</width>
    <height>445</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Form</string>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="5" column="1">
    <widget class="QCheckBox" name="checkBox_Tor">
     <property name="text">
      <string>Tor</string>
     </property>
    </widget>
   </item>
   <item row="0" column="1">
    <layout class="QVBoxLayout" name="verticalLayout_2">
     <item>
      <widget class="QLabel" name="labelState">
       <property name="font">
        <font>
         <pointsize>14</pointsize>
        </font>
       </property>
       <property name="styleSheet">
        <string notr="true"/>
       </property>
       <property name="text">
        <string>Состояние: Выключен</string>
       </property>
      </widget>
     </item>
     <item>
      <layout class="QHBoxLayout" name="horizontalLayout">
       <item>
        <widget class="QPushButton" name="pushButton_StartStop">
         <property name="text">
          <string>Старт</string>
         </property>
        </widget>
       </item>
       <item>
        <spacer name="horizontalSpacer_2">
         <property name="orientation">
          <enum>Qt::Horizontal</enum>
         </property>
         <property name="sizeHint" stdset="0">
          <size>
           <width>40</width>
           <height>20</height>
          </size>
         </property>
        </spacer>
       </item>
      </layout>
     </item>
    </layout>
   </item>
   <item row="11" column="1">
    <widget class="QLabel" name="label_2">
     <property name="text">
      <string>Параметры поиска:</string>
     </property>
    </widget>
   </item>
   <item row="10" column="1">
    <widget class="Line" name="line">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
    </widget>
   </item>
   <item row="1" column="1">
    <widget class="QLabel" name="label">
     <property name="text">
      <string>URL: </string>
     </property>
    </widget>
   </item>
   <item row="8" column="1">
    <widget class="QCheckBox" name="checkBox_Win">
     <property name="text">
      <string>Win</string>
     </property>
    </widget>
   </item>
   <item row="13" column="1">
    <widget class="QLineEdit" name="lineEdit_searchParametrs"/>
   </item>
   <item row="3" column="1">
    <widget class="QCheckBox" name="checkBox_noCast">
     <property name="text">
      <string>-no -cast</string>
     </property>
    </widget>
   </item>
   <item row="16" column="1">
    <spacer name="horizontalSpacer">
     <property name="orientation">
      <enum>Qt::Horizontal</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>40</width>
       <height>20</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="7" column="1">
    <widget class="QCheckBox" name="checkBox_RiskLvL">
     <property name="text">
      <string>Risk, LvL</string>
     </property>
    </widget>
   </item>
   <item row="0" column="0" rowspan="17">
    <widget class="QTextEdit" name="InformationTable"/>
   </item>
   <item row="15" column="1">
    <spacer name="verticalSpacer">
     <property name="orientation">
      <enum>Qt::Vertical</enum>
     </property>
     <property name="sizeHint" stdset="0">
      <size>
       <width>20</width>
       <height>40</height>
      </size>
     </property>
    </spacer>
   </item>
   <item row="4" column="1">
    <widget class="QCheckBox" name="checkBox_Command">
     <property name="text">
      <string>Command</string>
     </property>
    </widget>
   </item>
   <item row="6" column="1">
    <widget class="QCheckBox" name="checkBox_Hex">
     <property name="text">
      <string>-hex</string>
     </property>
    </widget>
   </item>
   <item row="2" column="1">
    <widget class="QLineEdit" name="lineEdit_url"/>
   </item>
  </layout>
 </widget>
 <resources/>
 <connections/>
</ui>
